<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GD_QLDuAn</class>
 <widget class="QWidget" name="GD_QLDuAn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QFrame" name="topBarFrame">
     <layout class="QHBoxLayout" name="topBar">
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Tìm kiếm theo tên, mã dự án...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Làm mới</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/refresh.svg</normaloff>:/icons/refresh.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="actionFrame">
     <layout class="QHBoxLayout" name="actionLayout">
      <item>
       <widget class="QPushButton" name="btnAddProject">
        <property name="text">
         <string>Thêm Dự Án</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEditProject">
        <property name="text">
         <string>Sửa</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteProject">
        <property name="text">
         <string>Xóa</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/delete.svg</normaloff>:/icons/delete.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDetailProject">
        <property name="text">
         <string>Chi Tiết</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/detail.svg</normaloff>:/icons/detail.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCompleteProject">
        <property name="text">
         <string>Hoàn Thành</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/check.svg</normaloff>:/icons/check.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableProjects">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Mã DA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tên Dự Án</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Người Phụ Trách</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ngày Bắt Đầu</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Trạng Thái</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Số Lượng TV</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>